homeassistant/components/satel_integra.py

```diff
@@ -101,7 +106,8 @@
 
     task_zones = hass.async_create_task(
         async_load_platform(hass, 'binary_sensor', DOMAIN,
-                            {CONF_ZONES: zones}, config))
+                            {CONF_ZONES: zones, CONF_OUTPUTS: outputs}, config)
+        )
 
     await asyncio.wait([task_control_panel, task_zones], loop=hass.loop)
```

Comment:
We shouldn't wait for these tasks. That could lead to deadlock.

Reply:
This hasn't been touched in this PR. Also, no deadlock is not happening in the field. Can we then agree to put in bug for it and fix it within separate PR?

Reply:
That's ok.

Reply:
Issue created: https://github.com/home-assistant/home-assistant/issues/19329